Logger/BaseLoggerMixins.cs

```diff
@@ -1,6 +1,39 @@
-﻿namespace Logger;
+using System;
+namespace Logger;
 
 public static class BaseLoggerMixins
 {
-    
+    public static void Error(this BaseLogger baseLogger, string message, params object[] args)
+    {
+        if (baseLogger is null) { throw new ArgumentNullException(); }
+
+        LogLevel logLevel = LogLevel.Error;
+        baseLogger.Log(logLevel, string.Format(message, args));
+    }
+
+    public static void Warning(this BaseLogger baseLogger, string message,
+                               params object[] args)
+    {
+        if (baseLogger is null) { throw new ArgumentNullException(); }
+
+        LogLevel logLevel = LogLevel.Warning;
+        baseLogger.Log(logLevel, string.Format(message, args));
+    }
+
+    public static void Information(this BaseLogger baseLogger, string message,
```

Comment:
```suggestion
    public static void Information(this BaseLogger logger, string message,
```
I'd recommend calling this 'logger' as your BaseLogger is a base class with lots of children that can be inheriting from it, so, this could actually be a FileLogger for example.